sequence_processing_pipeline/tests/test_commands.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+from unittest.mock import patch
+import tempfile
+import gzip
+import os
+from sequence_processing_pipeline.Commands import (split_similar_size_bins,
+                                                   demux)
+import io
+
+
+class CommandTests(unittest.TestCase):
+    @patch('os.stat')
+    @patch('glob.glob')
+    def test_split_similar_size_bins(self, glob, stat):
+        class MockStat:
+            st_size = 2 ** 28  # 256MB
+
+        mockglob = ['/foo/bar/a_R1_.fastq.gz',
```

Comment:
Would the underscore beside the period cause any issues? Another way to ask this, would the file names need to end on an underscore for things to work fine? Not sure, just asking.

Reply:
No, see https://github.com/biocore/mg-scripts/pull/105/files/e08f8d03a03a57326b31d564fc2d70cd508083ed#diff-e5c02a04eacc2aabc126515459eee73ff33eff7a89ac4999f3692d3320375e68R6